public/locales/en/translation.json

```diff
@@ -316,6 +316,13 @@
     },
     "contactUsModal": {
       "message": "Thank you for leaving us message!"
+    },
+    "contactUsInfo": {
+      "header": "Leave us a message",
+      "description": "Torquatos nostros? quos tu tam crudelis fuisse, nihil oportere nimium nos causae confidere, sed animo etiam erga nos causae confidere, sed uti oratione perpetua malo quam ostendis sed quia dolor repellendus temporibus autem.",
```

Comment:
is this placeholder text? i think its a little late, it detracts from your demo. can you put something real here?

Reply:
Done